script/bootstrap

```diff
@@ -106,6 +106,20 @@ install_packages_apt()
         fi
     }
 
+    # dhcpv6 pd
+    with DHCPV6_PD_REF && if [[ $PLATFORM == "raspbian" ]]; then
+        sudo apt-get install --no-install-recommends -y dhcpcd
+        sudo mkdir -p /etc/systemd/system/dhcpcd.service.d
+        # allow edit system config file eg. /etc/radvd.conf
+        sudo tee /etc/systemd/system/dhcpcd.service.d/custom.conf >/dev/null <<EOF
+[Service]
+ProtectSystem=false
```

Comment:
This didn't work. It needs further investigation.

Reply:
can change the whole service file, refer to the design doc [go/ot-ref-pd](http://goto.google.com/ot-ref-pd)

Reply:
The root cause is that full raspi OS release for Raspi 5 is signatured "debian" instead of "raspian"